cleanlab/count.py

```diff
@@ -732,8 +735,14 @@ def estimate_confident_joint_and_cv_pred_proba(
         clf_copy = sklearn.base.clone(clf)
 
         # Select the training and holdout cross-validated sets.
-        X_train_cv, X_holdout_cv = X[cv_train_idx], X[cv_holdout_idx]
-        s_train_cv, s_holdout_cv = labels[cv_train_idx], labels[cv_holdout_idx]
+        s_train_cv, s_holdout_cv = (
+            labels[cv_train_idx],
+            labels[cv_holdout_idx],
+        )  # labels are always np.array
```

Comment:
The comment was intended for this line, it doesn't make sense to explain labels_to_array() since the function name is already self-explanatory?

Reply:
If the goal is to guarantee this is true at this place in the code, you could assert(type(x) == np.ndarray)

Reply:
the goal is to explain that we don't need to consider ``labels.iloc[cv_train_idx]`` because we already know it is array not pandas. Added clarification.